lib/charms/layer/jenkins/configuration.py

```diff
@@ -99,3 +100,16 @@ def _set_prefix(self, prefix):
             return True
 
         return False
+
+    def set_update_center_ca(self):
+        """Configure Jenkins Update Center CA cert"""
+        config = hookenv.config()
+        ca_cert = config["update-center-ca"]
+        ca_cert_file = os.path.join(paths.UPDATE_CENTER_ROOT_CAS,
+                                    "default.crt")
+        host.mkdir(paths.UPDATE_CENTER_ROOT_CAS, perms=0o755)
+        host.write_file(ca_cert_file, ca_cert, perms=0o755)
+        host.chownr(ca_cert_file,
+                    owner="jenkins",
+                    group="jenkins",
+                    chowntopdir=True)
```

Comment:
`host.mkdir` and `host.write_file` both seem to support `owner` and `group` arguments, so would be cleaner to use those.